dist/dist-packages/linux/openziti-router/bootstrap.bash

```diff
@@ -69,13 +69,15 @@ enroll() {
   
   if [[ -n "${1:-}" && ! "${1}" =~ ^-- ]]; then
     local _config_file="${1}"
+    local _ziti_home
+    _ziti_home="$(dirname "${_config_file}")"
     shift
   else
-    echo "ERROR: no config file path provided" >&2
+    echo "ERROR: no config file path provided in first param" >&2
     return 1
   fi
 
-  if [[ ! -s "${ZITI_HOME}/${ZITI_ROUTER_IDENTITY_CERT}" || "${1:-}" == --force ]]; then
+  if [[ ! -s "${_ziti_home}/${ZITI_ROUTER_IDENTITY_CERT}" || "${1:-}" == --force ]]; then
```

Comment:
in general, this line looks wrong to me now that it's singled out. i would not expect this to be prepended with any path. it should just be `${ZITI_ROUTER_IDENTITY_CERT}`.

Reply:
I found another way that doesn't require this to be a relative path.

Reply:
I found another way that doesn't require this to be a relative path.